rector.php

```diff
@@ -89,9 +99,4 @@
     $rectorConfig->rule(FuncGetArgsToVariadicParamRector::class);
     $rectorConfig->rule(MakeInheritedMethodVisibilitySameAsParentRector::class);
     $rectorConfig->rule(SimplifyEmptyArrayCheckRector::class);
-    $rectorConfig
-        ->ruleWithConfiguration(TypedPropertyFromAssignsRector::class, [
-            // Set to false if you use in libraries, or it does create breaking changes.
-            TypedPropertyFromAssignsRector::INLINE_PUBLIC => false,
-        ]);
```

Comment:
this can be changed to `$rectorConfig->rule(TypedPropertyFromAssignsRector::class);` since default inline_public is false.

Reply:
Updated.